### PR TITLE
Configure local play bar

### PR DESCRIPTION
**Pull request by MyNameIsJonathan**

### CODE REVIEW

**Comment by MyNameIsJonathan** [comment]
Implemented a script that will calculate 10M songs, 50M likes, and 25K song play history
Created schema files for both PostgreSQL and Cassandra; both schemas successfully create requisite tables, insert into tables, and print from tables

File: database/data/generateData.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  //       console.log(err);
+  //     }
+  //   });
+  // }
+
+  // GENERATE AND SAVE LIKE EXAMPLES
+  for (let i = 0; i < 50000000; i += 1) {
+    // get a song id < 10,000,000
+    const songId = faker.random.number({
+      min: 0,
```
Comment (angeliquemari) [inline database/data/generateData.js]: didn't we say we'd use song ID from 1 - 1,000,000?

File: database/data/generateData.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  // }
+};
+
+writeHeaders();
+
+const start = Date.now();
+generateAndSaveData();
+const end = Date.now();
+
+console.log('total time:', end - start);
```
Comment (angeliquemari) [inline database/data/generateData.js]: handy!

File: database/postgreSQL/schema.sql [hunk 3]
```diff
@@ -2,32 +2,72 @@ DROP DATABASE IF EXISTS playbar;
...
 
-USE playbar;
+\c playbar;
 
-CREATE TABLE songs (
-  songId INT AUTO_INCREMENT,
-  likeCount INT DEFAULT 0,
+CREATE TABLE songs 
+(
+  songId serial PRIMARY KEY,
```
Comment (angeliquemari) [inline database/postgreSQL/schema.sql]: just fyi this will be converted to lowercase in your db

File: database/postgreSQL/schema.sql [hunk 4]
```diff
@@ -2,32 +2,72 @@ DROP DATABASE IF EXISTS playbar;
...
+CREATE TABLE playHistory 
+(
+  id serial,
+  songId integer,
+  userName VARCHAR(30),
   FOREIGN KEY (songId)
     REFERENCES songs(songId)
+);
+
+INSERT INTO songs (
```
Comment (angeliquemari) [inline database/postgreSQL/schema.sql]: assume you're going to remove the insertions / queries later

Comment (MyNameIsJonathan) [reply database/postgreSQL/schema.sql]: Yes, those are for testing and will be removed for future use

File: database/data/generateData.js [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+  //   // write song array to file
+  //   fs.appendFileSync(songsCSV, song, (err) => {
+  //     if (err) {
+  //       console.log(err);
+  //     }
+  //   });
+  // }
+
+  // GENERATE AND SAVE LIKE EXAMPLES
+  for (let i = 0; i < 50000000; i += 1) {
```
Comment (angeliquemari) [inline database/data/generateData.js]: nice!

**Pull request merged**